### PR TITLE
[FIX] apropriate call _get_reconcilation_methods

### PR DESCRIPTION
**Pull request by i-vyshnevska**
fix call to parent method, fix forgotten test import

### CODE REVIEW

**Comment by pedrobaeza** [comment]
It seems the test is not correct

File: account_mass_reconcile_transaction_ref/tests/test_account_reconcile_transaction_ref.py [hunk 1]
```diff
@@ -72,6 +72,7 @@ def test_mass_reconcile_transaction_ref_vs_ref(self):
         })
         count = reconcile.unreconciled_count
         reconcile.run_reconcile()
+        reconcile._get_total_unrec()
```
Comment (grindtildeath) [inline account_mass_reconcile_transaction_ref/tests/test_account_reconcile_transaction_ref.py]: I'm not sure about this one, IMO it's an issue with the dependencies of `_get_total_unrec` : https://github.com/OCA/account-reconcile/blob/11.0/account_mass_reconcile/models/mass_reconcile.py#L119

Maybe we should add `history_ids` to make sure the field is recomputed after `run_reconcile` ?

Comment (i-vyshnevska) [reply account_mass_reconcile_transaction_ref/tests/test_account_reconcile_transaction_ref.py]: added

**Review by grindtildeath — APPROVED** [review]
LGTM :+1:

**Review by tschanzt — APPROVED** [review]

**Comment by i-vyshnevska** [comment]
@yvaucher can we try merge this one again

**Pull request merged**